packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -110,7 +110,7 @@ abstract final class CupertinoColors {
   /// or the foreground red theme in various native apps such as HealthKit.
   ///
   /// This is SystemRed in the iOS palette.
-  static const Color destructiveRed = systemRed;
+  static const CupertinoDynamicColor destructiveRed = systemRed;
```

Comment:
Pinging @LongCatIsLooong, do you know if leaving this as a `Color` type rather than a `CupertinoDynamicColor` was intended? It's a PR from ages ago, but it looks like the change to not just using a [static color was here](https://github.com/flutter/flutter/pull/40566/files#diff-6754567927981e2770580e4296e76f0c79742a601a8b15877751b2617588b4b9R95).